### PR TITLE
[FIX] fix call virtaual

### PR DESCRIPTION
**Pull request by Jim8y**
It is possible to compile unsupported syntax and cause error script. thus having this pr to fix it.

error NC2001: Unsupported syntax: object.ToString().

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert/CallHelpers.cs [hunk 1]
```diff
@@ -224,6 +224,8 @@ private void CallVirtual(IMethodSymbol symbol)
         IFieldSymbol[] fields = members.OfType<IFieldSymbol>().ToArray();
         IMethodSymbol[] virtualMethods = members.OfType<IMethodSymbol>().Where(p => p.IsVirtualMethod()).ToArray();
         int index = Array.IndexOf(virtualMethods, symbol);
+        if (index < 0)
+            throw new CompilationException(symbol, DiagnosticId.SyntaxNotSupported, $"Unsupported syntax: {symbol.OriginalDefinition}.");
```
Comment (Hecate2) [inline src/Neo.Compiler.CSharp/MethodConvert/CallHelpers.cs:228]: Could we just say `Cannot call virtual method {...}`?

Comment (Jim8y) [reply src/Neo.Compiler.CSharp/MethodConvert/CallHelpers.cs:228]: To be honest, i dont really know what this syntax means. though its virtualmethods, but i have no idea why and how these code works.

Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert/CallHelpers.cs:228]: where did you get this error?

Comment (Jim8y) [reply src/Neo.Compiler.CSharp/MethodConvert/CallHelpers.cs:228]: object.tostring

**Review by shargon — APPROVED** [review]

**Pull request merged**